src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -28,41 +34,86 @@
 
 class GenomeStatus:
 
-    def __init__(self, original_id, resolved_id, attempted=False, filename=""):
+    def __init__(self,
+                 original_id,
+                 resolved_refseq_id,
+                 attempted=False,
+                 filename=""):
         self.original_id = ';'.join(original_id.split(','))
-        self.resolved_id = None if resolved_id == 'None' else resolved_id
-        self.attempted = True if attempted == 'True' else False
+        self.resolved_refseq_id = None if resolved_refseq_id == 'None' else resolved_refseq_id
+        if ast.literal_eval(attempted):
```

Comment:
Why do you use `ast.literal_eval` here?

Reply:
I thought it was nicer, but I can change it back to "True" str thing

Reply:
The original code is better `self.attempted = True if attempted == 'True' else False`, which is the typical way to convert string to boolean.